src/main/webapp/scripts/navbar.js

```diff
@@ -0,0 +1,31 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+/**
+ * Function to load the navbar on the web app 
+ */
+
+$(() => {
+  $('#navbar_container').load('common/navbar.html',setActiveTab);
```

Comment:
Nit: (styleguide) whitespace after comma
```suggestion
  $('#navbar_container').load('common/navbar.html', setActiveTab);
```